ocaml/idl/datamodel_host.ml

```diff
@@ -1745,6 +1745,13 @@ let set_https_only =
       ]
     ~allowed_roles:_R_POOL_OP ()
 
+let save_rrds =
```

Comment:
Why add this to the high-level API and xe CLI if we are just going to call the rrd-cli from the bugtool?